python/lsst/pipe/tasks/processCcdWithFakes.py

```diff
@@ -56,11 +64,15 @@ class ProcessCcdWithFakesConnections(PipelineTaskConnections,
         dimensions=("instrument", "visit", "detector")
     )
 
-    fakeCat = cT.Input(
-        doc="Catalog of fake sources to draw inputs from.",
+    fakeCats = cT.Input(
+        doc="Set of catalogs of fake sources to draw inputs from. We will "
+            "from one of these that fully contains our exposure or the is "
+            "closest the visit boresight.",
```

Comment:
@morriscb I guess the description here has a word missing between L68 and L69. L69 has `the is`. I believe the description also does not reflect the fact that it gathers fake sources from other neighboring tracts as done in `composeFakeCat`.

Reply:
As above, the doc string here doesn't read well - can you rephrase?